### DISCUSSION TITLE
Quiet mode for unit testing

### DISCUSSION THREAD

**Discussion by TaeyoonKwon** [rwf2/Rocket#2105] [Questions] [answered]
I am curious to know is there a **quiet mode** while `cargo test -- --nocapture`?

Whenever the server starts, I see bunch of configuration outputs, and I prefer having them while `cargo run`

however, I'd like to hide them while `cargo test -- --nocapture` so it does not show while I do the ci.

I am aware that `cargo test` will simply return ok when they are all ok, but it does not should printlns inside test codes.

Is there an option for it?

**Comment by wezm** [comment, marked as answer]
You can set the log level to `off` in your tests by overiding the configuration as shown in this comment:

https://github.com/SergioBenitez/Rocket/issues/1838#issuecomment-904513028
